Assets/MixedRealityToolkit-Tests/Scenes/BasicControllerView/01-Basic.unity

```diff
@@ -0,0 +1,482 @@
+--- !u!4 &1658344621
+Transform:
+  m_ObjectHideFlags: 0
+  m_PrefabParentObject: {fileID: 0}
+  m_PrefabInternal: {fileID: 0}
+  m_GameObject: {fileID: 1658344620}
+  m_LocalRotation: {x: 0, y: 0, z: 0, w: 1}
+  m_LocalPosition: {x: 0, y: 0, z: 0}
+  m_LocalScale: {x: 1, y: 1, z: 1}
+  m_Children: []
+  m_Father: {fileID: 1941297680}
+  m_RootOrder: 0
+  m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
+--- !u!114 &1658344622
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_PrefabParentObject: {fileID: 0}
+  m_PrefabInternal: {fileID: 0}
+  m_GameObject: {fileID: 1658344620}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 1077351063, guid: f70555f144d8491a825f0804e09c671c, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  m_HorizontalAxis: Horizontal
+  m_VerticalAxis: Vertical
+  m_SubmitButton: Submit
+  m_CancelButton: Cancel
+  m_InputActionsPerSecond: 10
+  m_RepeatDelay: 0.5
+  m_ForceModuleActive: 0
+--- !u!114 &1658344623
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_PrefabParentObject: {fileID: 0}
+  m_PrefabInternal: {fileID: 0}
+  m_GameObject: {fileID: 1658344620}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: -619905303, guid: f70555f144d8491a825f0804e09c671c, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  m_FirstSelected: {fileID: 0}
+  m_sendNavigationEvents: 1
+  m_DragThreshold: 10
+--- !u!20 &1658344624
+Camera:
+  m_ObjectHideFlags: 0
+  m_PrefabParentObject: {fileID: 0}
+  m_PrefabInternal: {fileID: 0}
+  m_GameObject: {fileID: 1658344620}
+  m_Enabled: 0
+  serializedVersion: 2
+  m_ClearFlags: 3
+  m_BackGroundColor: {r: 0.19215687, g: 0.3019608, b: 0.4745098, a: 0}
+  m_NormalizedViewPortRect:
+    serializedVersion: 2
+    x: 0
+    y: 0
+    width: 1
+    height: 1
+  near clip plane: 0.1
+  far clip plane: 1000
+  field of view: 60
+  orthographic: 1
+  orthographic size: 0.5
+  m_Depth: 0
+  m_CullingMask:
+    serializedVersion: 2
+    m_Bits: 4294967295
+  m_RenderingPath: -1
+  m_TargetTexture: {fileID: 0}
+  m_TargetDisplay: 1
+  m_TargetEye: 3
+  m_HDR: 0
+  m_AllowMSAA: 0
+  m_AllowDynamicResolution: 0
+  m_ForceIntoRT: 0
+  m_OcclusionCulling: 0
+  m_StereoConvergence: 10
+  m_StereoSeparation: 0.022
+--- !u!1 &1941297679
+GameObject:
+  m_ObjectHideFlags: 0
+  m_PrefabParentObject: {fileID: 0}
+  m_PrefabInternal: {fileID: 0}
+  serializedVersion: 5
+  m_Component:
+  - component: {fileID: 1941297680}
+  - component: {fileID: 1941297683}
+  - component: {fileID: 1941297682}
+  - component: {fileID: 1941297681}
+  - component: {fileID: 1941297685}
+  - component: {fileID: 1941297684}
+  m_Layer: 0
+  m_Name: Head
+  m_TagString: MainCamera
+  m_Icon: {fileID: 0}
+  m_NavMeshLayer: 0
+  m_StaticEditorFlags: 0
+  m_IsActive: 1
+--- !u!4 &1941297680
+Transform:
+  m_ObjectHideFlags: 0
+  m_PrefabParentObject: {fileID: 0}
+  m_PrefabInternal: {fileID: 0}
+  m_GameObject: {fileID: 1941297679}
+  m_LocalRotation: {x: -0, y: -0, z: -0, w: 1}
+  m_LocalPosition: {x: 0, y: 1, z: -10}
```

Comment:
The controller position is way off into space because the Head object is at (0, 1, -10). Updating this to (0, 0, 0), as well as reducing the cube to a MUCH smaller size haha ((0.1, 0.1, 0.1) worked fine), the controllers show up correctly.

Moving to (0, 0, 0) is a temporary fix though, as we'll need to take into account an object on top of the camera moving it, via teleportation or something similar.

In master, we use [`CameraCache.Main.transform.parent.TransformPoint`](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/master/Assets/HoloToolkit/Input/Scripts/InputSources/InteractionInputSource.cs#L737) to take this into account.

Reply:
Nice find, Kurtis (aka @keveleigh)!

Reply:
Now with gif:

![](https://user-images.githubusercontent.com/3580640/40944057-39cbe808-6808-11e8-8b38-2e007629eacd.gif)

Reply:
Great find @keveleigh 
In the Controller code it is transforming using ``` CameraCache.Main.transform.parent.TransformPoint```

If you look in 
https://github.com/SimonDarksideJ/MixedRealityToolkit-Unity/blob/867acc556d5e4b1b827e2dd18faf8e3e29ef60a3/Assets/MixedRealityToolkit/_Core/Devices/MixedReality/WindowsMixedRealityController.cs#L189

You'll see that is transformed already.  Obviously the default setup I created was slightly flawed.
So we'll need to think of a generic way to correct this automatically for projects.

Don't recall ever seeing any such guidance regarding scale for creating custom hands, so we'll likely need some docs on it.  The Cube's I initially created we the standard cube (e.g 1m, which might explain their size :D)

Great find and glad it was something so simple.

Reply:
@SimonDarksideJ Ahh yep, I realized after I commented that it actually isn't the teleportation issue I as I originally thought (especially since you already took that into account!), but actually an issue with the camera itself starting at a non-(0,0,0) location. I just checked, and we have this same problem in master too.

Reply:
That's partly why I wanted to use what we already have, but I'm glad it's working now.

Reply:
> I wanted to use what we already have

Have = have in master or somewhere else?

Reply:
Just confirmed and replicated locally.  Noticed a few interesting points we'll have to educate devs about, such as visualizer position (seems centered on the Touchpad), camera settings, etc.

It also highlights the need for:
* An SDK component to create a default "Rig"
* A Rig validator, to avoid the issues detected here.
Plus some other stuff, going to create some tasks for "next steps" once this is merged

Reply:
I meant the `InteractionInputSources` class. But @SimonDarksideJ has assured me we will do an A/B test to make sure nothing falls between the cracks.

Reply:
>such as visualizer position (seems centered on the Touchpad)

Assuming we're using the pointer position, it's centered at the tiny sphere here:
![ezgif-2-3ac46a4ce7](https://user-images.githubusercontent.com/3580640/41072414-7f056812-69b2-11e8-8638-81ab4b278999.gif)

Reply:
Real question is how'd you get the Samsung device to show up?